distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Table_Helpers.enso

```diff
@@ -500,10 +500,8 @@ resolve_order_by internal_columns column_selectors problem_builder =
             matches = match_columns name case_sensitivity internal_columns
             if matches.is_empty then problem_builder.report_missing_input_columns [name]
             matches.map c-> Column_Transform_Element.Value c selector
-    selectors_vec = case column_selectors of
-        _ : Vector -> column_selectors
-        _ -> [column_selectors]
-    selectors_vec.flat_map resolve_selector
+        _ : Function -> resolve_selector (selector:Sort_Column)
```

Comment:
Hmm I'm surprised this is needed.
Wouldn't it be enough to annotate `selector` above with type `selector : Integer | Text | Sort_Column`?

The `_ : Function` trick is doing very low-level internal stuff to get autoscoping to work. I think if possible, we should try to rely on the higher level type annotations to do all the work.